### PR TITLE
Update latex.md and add screenshots

### PR DESCRIPTION
**Pull request by rolandlo**
I have updated the content, referred to the improvements by @personalizedrefrigerator and added some more examples how to configure the template. Everything is accompanied with screenshots, which make the page much more visual.

### CODE REVIEW

File: docs/guide/tools/latex.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
 
-Using the LaTeX tool you can insert LaTeX formulas into a Xournal++ document. In order to use this tool you must have LaTeX installed on your system.
+Using the LaTeX tool you can insert LaTeX formulas into a Xournal++ document.
+In order to use this tool you must have LaTeX installed on your system.
+
+![LaTeX-Tool-Overview](../../img/guide/LaTeX_tool_overview.png)
+
+In the latest development version, the preview dialog is resizable, there is word
+wrap, auto-indentation, syntax highlighting and a separate `Command Output` tab,
+where the feedback of your LaTeX generator (e.g. `pdflatex`) is displayed.
```
Comment (Technius) [inline docs/guide/tools/latex.md:10]: I would wrap this in a note:

```suggestion
!!! note
    The screenshots on this page show the latest nightly version.
    In the latest nightly version, the preview dialog is has a resizable
    window, word wrap, auto-indentation, and syntax highlighting.
    It also has a separate `Command Output` tab that displays the output from
    the LaTeX generator command (e.g. `pdflatex`).
```

File: docs/guide/tools/latex.md [hunk 2]
```diff
@@ -58,21 +86,26 @@ The LaTeX formulas are rendered by substituting the user-provided formula (along
...
 ```
 
+![LaTeX-Rotated-Text](../../img/guide/LaTeX_rotated_text.png)
+
 - Include the [**TikZ** package](https://www.ctan.org/pkg/pgf) via `\usepackage{tikz}` for creating drawings with the powerful TikZ tool. A simple example would be to draw a table (or grid) with n rows and m columns via `\tikz{\draw (0,0) grid(m,n)}`. See [this issue](https://github.com/xournalpp/xournalpp/issues/2179) for more information.
 
-- Include the [**PGFPlots** package](https://www.ctan.org/pkg/pgfplots) via `\usepackage{pgfplots}` for plotting function graphs. Using a macro you will be able to plot function graphs nicely and very quickly. For example, add the following lines to your template:
+![LaTeX-TikZ-Table](../../img/guide/LaTeX_grid.png)
+
+- Include alongside TikZ the [**PGFPlots** package](https://www.ctan.org/pkg/pgfplots) via `\usepackage{pgfplots}`
```
Comment (Technius) [inline docs/guide/tools/latex.md:106]:
```suggestion
- With TikZ included, also include the [**PGFPlots** package](https://www.ctan.org/pkg/pgfplots) via `\usepackage{pgfplots}` 
```

File: docs/guide/tools/latex.md [hunk 3]
```diff
@@ -58,21 +86,26 @@ The LaTeX formulas are rendered by substituting the user-provided formula (along
...
 
+![LaTeX-Rotated-Text](../../img/guide/LaTeX_rotated_text.png)
+
 - Include the [**TikZ** package](https://www.ctan.org/pkg/pgf) via `\usepackage{tikz}` for creating drawings with the powerful TikZ tool. A simple example would be to draw a table (or grid) with n rows and m columns via `\tikz{\draw (0,0) grid(m,n)}`. See [this issue](https://github.com/xournalpp/xournalpp/issues/2179) for more information.
 
-- Include the [**PGFPlots** package](https://www.ctan.org/pkg/pgfplots) via `\usepackage{pgfplots}` for plotting function graphs. Using a macro you will be able to plot function graphs nicely and very quickly. For example, add the following lines to your template:
+![LaTeX-TikZ-Table](../../img/guide/LaTeX_grid.png)
+
+- Include alongside TikZ the [**PGFPlots** package](https://www.ctan.org/pkg/pgfplots) via `\usepackage{pgfplots}` 
+for plotting function graphs. Using a macro you will be able to plot function graphs nicely and very quickly.
```
Comment (Technius) [inline docs/guide/tools/latex.md:107]:
```suggestion
for plotting function graphs. Using a macro, you will be able to plot function graphs nicely and very quickly. 
```

**Review by Technius — APPROVED** [review]
The pictures are a really great addition!

File: docs/guide/tools/latex.md (hunk 1 above)
Comment (rolandlo) [reply docs/guide/tools/latex.md]: Great idea!

**Pull request merged**